packages/terra-base/docs/README.md

```diff
@@ -20,6 +20,41 @@ import Base from 'terra-base';
 </Base>
 ```
 
+## I18n
+
+The `locale` prop provided to `Base` will be used to loaded the appropriate translation messages. `Base` additionally wraps its children with an `I18nProvider` component (as provided by `terra-i18n`) that exposes the loaded messages to its children through its context.
```

Comment:
*load the appropriate translation messages.

Reply:
Updated: https://github.com/cerner/terra-core/pull/1379/commits/68616339486b3447a8a0f9e297f2f6cde08eebad